config/environments/sandbox.rb

```diff
@@ -125,29 +125,6 @@
 
   # Logging
   config.log_level = :info
-  config.log_tags = [:request_id] # Prepend all log lines with the following tags.
-  logger = ActiveSupport::Logger.new($stdout)
-  logger.formatter = config.log_formatter
-  config.logger = ActiveSupport::TaggedLogging.new(logger)
-  config.active_record.logger = nil # Don't log SQL in production
-
-  # Use Lograge for cleaner logging
-  config.lograge.enabled = true
-  config.lograge.base_controller_class = ["ActionController::API", "ActionController::Base"]
-  config.lograge.formatter = Lograge::Formatters::Logstash.new
-  config.lograge.ignore_actions = ["ApplicationController#check"]
-  config.lograge.logger = ActiveSupport::Logger.new($stdout)
-
-  # Include params in logs: https://github.com/roidrage/lograge#what-it-doesnt-do
-  config.lograge.custom_options = lambda do |event|
-    exceptions = %w[controller action format id]
-    {
-      params: event.payload[:params].except(*exceptions),
-      exception: event.payload[:exception], # ["ExceptionClass", "the message"]
-      current_user_class: event.payload[:current_user_class],
```

Comment:
is there a way we can add back those params for the new logging, as we need them for the API logs?

Reply:
Don't we log all the API requests in a table anyway?

Happy for you to configure this as needed though.

Reply:
Unfortunately lograge is [listed as a conflicting gem](https://logger.rocketjob.io/rails.html#conflicting-gems) - happy to revisit this to tweak formats. Going to merge for now so we can at least get Logit back up and running.

Reply:
> Don't we log all the API requests in a table anyway?
Yes but we started sending everything to big query due to losing logit. We would like to have the good views logit gave us with this if we intend to set it back up to avoid having to use the weird dashboards we now have from BQ

Reply:
>Unfortunately lograge is [listed as a conflicting gem](https://logger.rocketjob.io/rails.html#conflicting-gems) - happy to revisit this to tweak formats. Going to merge for now so we can at least get Logit back up and running.

Sorry Im confused what does this have to do with adding custom fields, can we not add them with the new libraries?

Reply:
Sorry - I meant if it was a case of me just adding back that chunk of config I'd have done it now, but I suspect there's more to it.